<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetConfigurarDTX</class>
 <widget class="QWidget" name="WidgetConfigurarDTX">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="6">
    <widget class="QLabel" name="perfect_time_Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="speed_SpinBox">
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="speed_Label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save Configuration</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="good_Label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Good</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="great_Label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Great</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="theme_Label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Theme</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QLabel" name="good_time_Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="perfect_Label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Perfect</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QLabel" name="great_time_Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="exitButton">
     <property name="text">
      <string>Quit Configuration</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QLabel" name="poor_time_Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="theme_LineEdit"/>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QCheckBox" name="checkBox_Gl">
     <property name="text">
      <string>OpenGl</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="poor_Label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Poor</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="songs_Label">
     <property name="text">
      <string>Songs</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="songs_LineEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>saveButton</tabstop>
  <tabstop>exitButton</tabstop>
  <tabstop>speed_SpinBox</tabstop>
  <tabstop>checkBox_Gl</tabstop>
  <tabstop>theme_LineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
